minidump-common/src/format.rs

```diff
@@ -411,6 +413,26 @@ pub struct GUID {
     pub data4: [u8; 8],
 }
 
+impl fmt::Display for GUID {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
```

Comment:
`guid_to_string` already exists and probably there should only be one impl as this endianness is really hard to get right.

afaik this will print the "wrong" guids if the target is LE.  This is the same as breakpad does so makes sense for now.